parsl/tests/integration/test_parsl_load_default_config.py

```diff
@@ -0,0 +1,43 @@
+import parsl
```

Comment:
This test doesn't get run by the CI, but this is the sort of simple feature that should be tested there.

There's a pattern in `.travis.yml` in the root directory of the project, `parsl/tests/test*/test*` about which tests to run, and this test doesn't run because `/integration/` doesn't match `test*`.

I think you might need to make a test function called `test_*` inside this file, for pytest to discover how to run the test: https://docs.pytest.org/en/latest/goodpractices.html#test-discovery